### ISSUE TITLE
Change VENDOR_ID in QMK firmware for proper VIA compatibility

### ISSUE THREAD

**Issue by mpfl** [imchipwood/dumbpad#15]
Theoretically this is an issue to raise in the QMK repo, but I wanted to get your attention.

It looks like the VENDOR_ID for the Dumbpad is still the default of 0xFEED.

Would you be able to change the VENDOR_ID so that I can submit my VIA config to VIA? Alternatively, I would be happy to change the VENDOR_ID and submit a PR. Figure it would be polite to ask you first since you're the Dumbpad's creator!

**Comment by imchipwood** [comment]
Huh, thought I'd changed it to DEAF or something else silly.. Anyway thanks
for contacting me first - I don't really care what the vendor ID is so long
as the device ID still shows up as 'dumbpad' when plugged in!

Feel free to change it!

**Comment by imchipwood** [comment]
Looks like the qmk_firmware PR was reverted - are you planning on reworking that?

Anyway, I'm gonna close this - feel free to email me or contact me somewhere else if you want help with the VIA support, I've had a few folks ask if it's possible to add but haven't had the time to look into it.

**Comment by mpfl** [comment]
> Looks like the qmk_firmware PR was reverted - are you planning on reworking that?

Sorry, I get distracted easily and have been working on other projects. I might pick it up again but given how unreliable I know I am, I'll put it on hold for now.

**Comment by medoix** [comment]
Just noting that my PR to merge in the OLED firmware also makes the ```VENDOR_ID``` change here https://github.com/qmk/qmk_firmware/pull/17058/files#diff-c34bacd1d7271500b70bcbec925dae2da916c0e295de920decee8f4e66a67d78R22

This will then unblock the one current reason for no VIA support in this PR https://github.com/the-via/keyboards/pull/1255
